doc/varnish-configuration.rst

```diff
@@ -86,6 +98,25 @@ To enable this feature, add the following to ``your_varnish.vcl``:
 Read more on `handling PURGE requests`_ in the Varnish documentation (for
 Varnish 3, see `purging for Varnish 3`_).
 
+Purge with xkey
```

Comment:
to me this is essentially a tags topic, the fact that we "purge" the tag is the detail, not the main thing. so i would move this subsection below to the tags section. then we only have one place where we talk about xkey which should be less confusing.